pkgs/applications/graphics/ktikz/default.nix

```diff
@@ -8,18 +9,24 @@
 # See historical versions of this file for building ktikz with KDE4.
 
 stdenv.mkDerivation rec {
-  version = "unstable-20161122";
+  version = "0.12";
   name = "qtikz-${version}";
```

Comment:
Is it possible to use pname. Also is this the correct package name as a lot of things on the github page seem to be called ktikz.

Reply:
Thanks, will be updated.
As for the name, all I can do is quote https://github.com/fhackenberger/ktikz/blob/master/INSTALL
```

Install the Qt-only (qtikz) application (using qmake):
------------------------------------------------------
[...]

Install the KDE (ktikz) application and the KPart (using cmake):
----------------------------------------------------------------
```
the final binary is named qtikz.